src/Lesson4/Main.java

```diff
@@ -0,0 +1,18 @@
+package Lesson4;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        String str = "Sentences in upper and lower case with the output of the first and last letter";
+        String upperCase = str.toUpperCase();
+        String lowerCase = str.toLowerCase();
+        char firstLetter = str.charAt(0);
+        char lastLetter = str.charAt(77);
```

Comment:
why 77? we should use length() function. because we don't know the length of input string. Strings can be different

Reply:
Made changes as noted! Add a new commit to https://github.com/aqaman0011/myHomeworks/pull/4/commits/6467b3c2deb56d5f06e6ad516969b7cf7d429f2d
Can you revise my rating?